tests/rocksdb.rs

```diff
@@ -0,0 +1,95 @@
+use std::{
+    fmt::{Debug, Display},
+    fs,
+    sync::Arc,
+};
+use teloxide::{
+    dispatching::dialogue::{RocksDbStorage, RocksDbStorageError, Serializer, Storage},
+    types::ChatId,
+};
+
+#[tokio::test(flavor = "multi_thread")]
```

Comment:
Is there a particular reason to use `flavor = "multi_thread"`?

Reply:
Honestly, I have just copied tests from `tests/sqlite.rs`, but can't find any arguments for it. Should I remove it?